modules/installation-configuration-parameters.adoc

```diff
@@ -1084,7 +1089,11 @@ Additional GCP configuration parameters are described in the following table:
 |Parameter|Description|Values
 
 |`platform.gcp.network`
-|The name of the existing VPC that you want to deploy your cluster to.
+|The name of the existing VPC that you want to deploy your cluster to. If you want to deploy your cluster into a shared VPC, you must set `platform.gcp.networkProjectID` with the name of the GCP project that contains the shared VPC.
```

Comment:
VPC isn't spelled out anywhere. Consider spelling out the first instance before using an acronym.